### PR TITLE
Tests for ProtocolHandler Mailboxes (Neo 3.x)

### PR DESCRIPTION
**Pull request by igormcoelho**
Basic unit tests for Mailboxes.

### CODE REVIEW

**Comment by igormcoelho** [comment]
@vncoelho please apply your fix.

**Comment by vncoelho** [comment]
Brother, the fix was already applied.
Travis is strange...

**Comment by vncoelho** [comment]
These tests are not  easy to be made! Nice templates, @igormcoelho. Great job.

**Comment by igormcoelho** [comment]
I guess this is beautiful now. Can you guys approve it? @shargon @vncoelho

**Comment by shargon** [comment]
Where is the tests :P ?

File: neo.UnitTests/UT_ProtocolHandlerMailbox.cs [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+            uut.IsHighPriority(Message.Create(MessageCommand.Headers, s)).Should().Be(false);
+            uut.IsHighPriority(Message.Create(MessageCommand.GetBlocks, s)).Should().Be(false);
+            uut.IsHighPriority(Message.Create(MessageCommand.Mempool, s)).Should().Be(false);
+            uut.IsHighPriority(Message.Create(MessageCommand.Inv, s)).Should().Be(false);
+            uut.IsHighPriority(Message.Create(MessageCommand.GetData, s)).Should().Be(false);
+            uut.IsHighPriority(Message.Create(MessageCommand.NotFound, s)).Should().Be(false);
+            uut.IsHighPriority(Message.Create(MessageCommand.Transaction, s)).Should().Be(false);
+            uut.IsHighPriority(Message.Create(MessageCommand.Block, s)).Should().Be(false);
+            uut.IsHighPriority(Message.Create(MessageCommand.Consensus, s)).Should().Be(true);
+            uut.IsHighPriority(Message.Create(MessageCommand.Reject, s)).Should().Be(false);
```
Comment (igormcoelho) [inline neo.UnitTests/UT_ProtocolHandlerMailbox.cs]: Here the tests @shargon! hahaha 

Focused on ProtocolHandler this time... Coverage increased a lot already, and on future we cover others ;)

**Comment by igormcoelho** [comment]
Changed title now :P

**Comment by igormcoelho** [comment]
later I can try to.finish all.

**Comment by igormcoelho** [comment]
Dont accept yet

**Comment by igormcoelho** [comment]
Ok then, I have to do for 2x as well.. it will help Ricardo.

**Pull request merged**